### DISCUSSION TITLE
Release of 0.11.3

### DISCUSSION THREAD

**Discussion by AlexNigl** [fail2ban/fail2ban#3122] [Q&A] [answered]
According to [the latest Security Advisory](https://github.com/fail2ban/fail2ban/security/advisories/GHSA-m985-3f3v-cwmm) users of 0.11.2 can upgrade to version 0.11.3.
I couldn't find 0.11.3 on the [release page](https://github.com/fail2ban/fail2ban/releases) am I missing something?

**Comment by sebres** [comment, marked as answer]
It is not yet released **here** (in particular because of #3069), but the (hot-)fixes must be released on every affected distribution.

**Reply by AlexNigl** [reply]
Is there a timeline for the next release?
The last release is almost two years old and 190 commits behind.

**Reply by sebres** [reply]
I'll release v.1.0.1 soon, see also #3359
